scripts/run_script_e2e.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+# Prep for integration/e2e
+shout "Building odo binaries"
+make bin
+
+# copy built odo to GOBIN
+cp -avrf ./odo $GOBIN/
+shout "getting ginkgo"
+make goget-ginkgo
```

Comment:
I think `run_all_tests` script https://github.com/openshift/odo/blob/master/scripts/run_all_tests.sh is setting up similar stuffs. Shouldn't we consolidate the scripts?

Reply:
As it was discussed with @mohammedzee1000 that run_all_tests script will be divided into two different parts, i.e. setup scripts and run scripts, we will remove `run_all_tests` when the transition is complete. For now we are not removing run_all_tests because we are still using this script for Linux tests.